src/components/add/addForm.js

```diff
@@ -1,98 +1,97 @@
 import Icon from '@material-ui/core/Icon';
 import ShoppingCartIcon from '@material-ui/icons/ShoppingCart';
-import AddPhotoAlternate from '@material-ui/icons/AddPhotoAlternate'
+import AddPhotoAlternate from '@material-ui/icons/AddPhotoAlternate';
 import MenuItem from '@material-ui/core/MenuItem';
 // import Camera from './camera';
 import Camera, { FACING_MODES, IMAGE_TYPES } from 'react-html5-camera-photo';
 import 'react-html5-camera-photo/build/css/index.css';
 
-
-
-import ROUTE from '../../constants/route'
-import itemAction from "../../redux/actions/item"
+import ROUTE from '../../constants/route';
+import itemAction from '../../redux/actions/item';
 
 const styles = theme => ({
   root: {
     display: 'flex',
-    flexWrap: 'wrap',
+    flexWrap: 'wrap'
   },
   margin: {
-    margin: theme.spacing.unit,
+    margin: theme.spacing.unit
   },
   textField: {
     flexBasis: 200,
     backgroundColor: 'cornsilk'
   },
   button: {
-    backgroundColor: "seashell",
-    borderBottom: "solid 3px black",
-    borderLeft: "solid 3px darkgrey",
-    borderRight: "solid 3px darkgrey",
-    borderTop: "solid 3px lightgrey",
+    backgroundColor: 'seashell',
+    borderBottom: 'solid 3px black',
+    borderLeft: 'solid 3px darkgrey',
+    borderRight: 'solid 3px darkgrey',
+    borderTop: 'solid 3px lightgrey'
   }
 });
 
 const photos = [
   {
     value: './cupcakes',
-    label: 'Cupcakes',
+    label: 'Cupcakes'
   },
   {
     value: './Shovels',
-    label: 'Shovels',
+    label: 'Shovels'
   },
   {
     value: './lizards',
-    label: 'Lizards',
-  },
+    label: 'Lizards'
+  }
 ];
 
 class OutlinedInputAdornments extends React.Component {
   state = {
     name: '',
     picture: '',
     quantity: '',
+    soldCount: 0,
     price: ''
   };
 
   handleChange = prop => event => {
     this.setState({ [prop]: event.target.value });
   };
 
-  onTakePhoto (dataUri) {
-    this.setState({picture: dataUri});
+  onTakePhoto(dataUri) {
+    this.setState({ picture: dataUri });
     console.log('takePhoto');
   }
 
-  onCameraError (error) {
-  console.error('onCameraError', error);
-
-}
-
+  onCameraError(error) {
+    console.error('onCameraError', error);
+  }
 
   handleSave = () => {
-
-    if (this.state.name === '' || this.state.quantity === '' || this.state.price === ''){
-      alert("No changes saved.");
-      console.log("Leaving!");
+    if (this.state.name === '' || this.state.quantity === '' || this.state.price === '') {
+      alert('No changes saved.');
+      console.log('Leaving!');
     } else {
-      this.props.dispatch(itemAction.add({
-        name: this.state.name,
-        count: this.state.quantity,
-        price: this.state.price,
-        photo: this.state.picture
-      }))
+      this.props.dispatch(
+        itemAction.add({
+          name: this.state.name,
+          count: this.state.quantity,
+          price: this.state.price,
+          photo: this.state.picture,
```

Comment:
Use same definitions in the state as variables passed to the action. E.g. have picture or photo but not both